samples/oauth_php/VetsApiClaims/app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\OauthSession;
+
+class HomeController extends Controller
+{
+    public function getIndex()
+    {
+        $nonce_base = bin2hex(random_bytes(20));
+        session(['nonce_key' => $nonce_base]);
```

Comment:
just a preference, but, especially if you are using clean PHP 7 syntax and namespacing/injection etc.. may want to use the session objects in the request https://laravel.com/docs/5.8/session - there is little difference on a technical level - so, just a preference.